src/panels/OriginPanel/OriginPanel.tsx

```diff
@@ -53,13 +48,13 @@ export function OriginPanel() {
   useEffect(() => {
     // We make a copy here of the dispatchers to ensure that in the cleanup function they
     // will be exactly the same as they were on creation, i.e. not modifided
-    const anchor = anchorDispatcher.current;
-    const aim = aimDispatcher.current;
-    anchor.subscribe();
-    aim.subscribe();
+    const copyAnchorDispatcher = anchorDispatcher.current;
+    const copyAimDispatcher = aimDispatcher.current;
+    copyAnchorDispatcher.subscribe();
+    copyAimDispatcher.subscribe();
```

Comment:
Just wanted to rename `anchor` and `aim` so its less confusing that they are overriden in scope, not sure if the new name is much better though..